helm_deploy/offender-case-notes/templates/deployment.yaml

```diff
@@ -11,8 +11,8 @@ spec:
   strategy:
     type: RollingUpdate
     rollingUpdate:
-      maxSurge: 100%
-      maxUnavailable: 50%
+      maxSurge: 50%
+      maxUnavailable: 0
```

Comment:
prevent losing half the pods during new deployment and don't deploy all new pods at once